pkg/vmprovider/providers/vsphere/session/session_vm_customization_test.go

```diff
@@ -378,16 +378,27 @@ var _ = Describe("Cloud-Init Customization", func() {
 
 	Context("GetCloudInitPrepCustSpec", func() {
 		var (
-			err      error
 			custSpec *vimTypes.CustomizationSpec
 		)
 		JustBeforeEach(func() {
-			custSpec, err = session.GetCloudInitPrepCustSpec(cloudInitMetadata, updateArgs)
+			var (
+				err        error
+				configSpec *vimTypes.VirtualMachineConfigSpec
+			)
+			configSpec, custSpec, err = session.GetCloudInitPrepCustSpec(cloudInitMetadata, updateArgs)
+			Expect(err).ToNot(HaveOccurred())
+
+			// Validate that Cloud-Init Prep always uses the GuestInfo transport
+			// for the CI Prep's meta and user data.
+			Expect(configSpec).ToNot(BeNil())
+			Expect(configSpec.VAppConfig).ToNot(BeNil())
```

Comment:
tini nit： `vAppConfigSpec := configSpec.VAppConfig` for reuse
LGTM

Reply:
Thanks! I thought of that, but opt'd for uninterrupted Expect clauses.